.pre-commit-config.yaml

```diff
@@ -0,0 +1,29 @@
+repos:
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.29.1
+    hooks:
+    -   id: pyupgrade
+        args: [--py36-plus]
+  - repo: https://github.com/psf/black
+    rev: 21.9b0
+    hooks:
+      - id: black
```

Comment:
Is it possible to avoid adding this?

Reply:
It is, but why?

Reply:
Bad prior experience, gives me flashbacks to the point I try to avoid this thing/contribute to projects that try to throw it in my face.

Reply:
Hmm sorry to hear that. In my experience this is always beneficial, because both contributors and reviewers no longer need to worry about code formatting at all.

I personally prefer to have it, but I will let @RonnyPfannschmidt make the final call.

Reply:
anything on pytest-dev we use brings in black/pre-commit - so its pretty much our standard already - its not like there will be  any surprises afterwards unless one tries to break them and/or deliberately avoids running them